.github/workflows/pr-checks.yml

```diff
@@ -0,0 +1,60 @@
+name: PR checks
+on:
+  pull_request:
+env:
+  CI: true
+jobs:
+  test:
+    name: Test using JDK ${{matrix.java}} and Scala ${{matrix.scala}}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        java: [8, 11]
+        scala:
+          - 2.11.12
```

Comment:
Shouldn't we add all versions we release for?
i.e. on top of what you have I'd add:
- 2.13.0
- 2.12.10
- 2.12.9
- 2.12.8

Or should we skip releasing for these?

Reply:
We don't release for a patch version, you release for the minor version. It won't harm to test against more versions though.

Reply:
OK. I am not sure which are patch versions here.
And I took the version list from: https://search.maven.org/search?q=g:com.sksamuel.scapegoat%20scalac-scapegoat%201.4.1 as I thought it is was we release for.

Obv. it's up to you

Reply:
Yeah since 1.4.0 scapegoat uses full cross version (https://github.com/sksamuel/scapegoat/commit/d3633ac2624980831370d4bd160b9298eee7337f). If we could agree to targeting specific versions, I'll update this PR and also reflect this when automating the new release workflow.

- only the latest version of 2.11 since 2.11 is no longer maintained (unless we need to drop support for 2.11)
- two latest versions of both 2.12 and 2.13

Are you guys happy with the above?

Reply:
I don't have any strong preference as long as it's consistently applied (i.e. what we test for and what we release).
Your suggestion seems fine to me.